src/locales/fr/party-ui-handler.ts

```diff
@@ -4,8 +4,14 @@ export const partyUiHandler: SimpleTranslationEntries = {
   "ALL": "Tout",
   "SEND_OUT": "Envoyer",
   "SUMMARY": "Résumé",
-  "CANCEL": "Annuler",
+  "CANCEL": "Sortir",
```

Comment:
```suggestion
  "CANCEL": "Retour",
```

For the `cancelButton` I don't mind "Sortir" being used, but as far as this input is concerned, it's been "Retour" for several generations now.

Your change:
![Capture d’écran du 2024-06-17 14-04-41](https://github.com/pagefaultgames/pokerogue/assets/8146474/df0ddc82-eed8-4667-9058-165a4ecc8fa4)

In official French games:
![Screenshot_20240617_143137_SuperNDS](https://github.com/pagefaultgames/pokerogue/assets/8146474/aedd54e6-9980-450a-bc9f-8f7f765327e0)